content/en/getting-started/configuration.md

```diff
@@ -77,6 +77,27 @@ foo = "bar"
 ```
 
 Considering the structure above, when running `hugo --environment staging`, Hugo will use every settings from `config/_default` and merge `staging`'s on top of those.
+
+Let's take an example to understand this better. Let's say you are using Google analytics for your website. This requires you to specify `googleAnalytics = "UA-XXXXXXXX"` in `config.toml`. Now consider the following scenario:
```

Comment:
**Google analytics** should be **Google Analytics**

Reply:
Fixed this